refresh/config.go

```diff
@@ -24,7 +26,13 @@ type Configuration struct {
 }
 
 func (c *Configuration) FullBuildPath() string {
-	return path.Join(c.BuildPath, c.BinaryName)
+	path := path.Join(c.BuildPath, c.BinaryName)
```

Comment:
`path` may not be the best variable name, given the use of the path package as well (a bit ambiguous)